src/binary/non_blocking/binary_buffer.rs

```diff
@@ -690,7 +690,7 @@ mod tests {
         let mut buffer = BinaryBuffer::new(&[0b1000_0000]);
         let var_int = buffer.read_uint(buffer.remaining())?;
         assert_eq!(var_int.size_in_bytes(), 1);
-        assert_eq!(var_int.value(), &UInt::from(128));
+        assert_eq!(var_int.value(), &UInt::from(128u64));
```

Comment:
🗺️ Signed integers can no longer be infallibly converted to a `UInt`. In the absence of other type information, Rust treats integers as `i32`s. This means that to call `UInt::from(...)`, we need to mark our integers as an unsigned type.